iotdb-core/datanode/src/main/java/org/apache/iotdb/db/utils/MmapUtil.java

```diff
@@ -32,4 +33,11 @@ public static void clean(MappedByteBuffer mappedByteBuffer) {
     }
     PlatformDependent.freeDirectBuffer(mappedByteBuffer);
   }
+
+  /** we do not need to clean heapByteBuffer manually, so we just leave it alone. */
+  public static void clean(ByteBuffer byteBuffer) {
+    if (byteBuffer != null & byteBuffer instanceof MappedByteBuffer) {
```

Comment:
Bitwise operator `&` is a little bit weird here.

Reply:
Nice catch!
BTW, IDEA reminds me that the later null instanceof any class will be false, so the previous judgement is not needed